src/components/profile/profile-test.stories.tsx

```diff
@@ -1,10 +1,9 @@
-/* eslint-disable react/prop-types */
 import React from "react";
 
 import specialCharacters, {
   email as testEmail,
 } from "../../__internal__/utils/argTypes/specialCharacters";
-import Profile from "./profile.component";
+import Profile, { ProfileProps } from "./profile.component";
```

Comment:
It might be useful to use `import { ComponentStory, ComponentMeta } from '@storybook/react';` here but it's not essential

https://storybook.js.org/docs/react/writing-stories/introduction#defining-stories

Reply:
thanks, wasn't aware of that. I'll leave it as I prefer to only add types which are either needed to avoid errors/warnings, or those which are definitely helpful as documentation. These types don't appear to be needed for the first of these (not sure why tbh but there you go) and I don't personally think they add much that's useful.